src/utils/console.js

```diff
@@ -0,0 +1,17 @@
+import consola from 'consola';
+import colors from 'colors';
```

Comment:
If colours are used only on dev, then this will throw an error in prod I suppose?

Reply:
ya fair point. I removed colors completely and now we are using winston. Winston is added in dependencies